### PR TITLE
Add clarification for #142

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Add clarification for #142

... and couple cosmetic changes

### CODE REVIEW

File: trace_context/HTTP_HEADER_FORMAT.md [hunk 1]
```diff
@@ -144,12 +149,37 @@ boolean traced = (traceFlags & FLAG_TRACED) == FLAG_TRACED
 
 #### Requested Flag (00000001)
```
Comment (codefromthecrypt) [inline trace_context/HTTP_HEADER_FORMAT.md:150]: I think this should be renamed to sampled as that's far more common usage

File: trace_context/HTTP_HEADER_FORMAT.md [hunk 2]
```diff
@@ -144,12 +149,37 @@ boolean traced = (traceFlags & FLAG_TRACED) == FLAG_TRACED
...
+
+Caller who defers a tracing decision SHOULD leaves this flag unset. Caller may
+communicate the priority of deferred decision (most likely will be collected)
+using `recorded` flag.
+
+Flag `recorded` can also be used to communicate the ask to collect information
+or notify that information was collected and available for querying from the
+caller service.
 
 #### Recorded Flag (00000010)
```
Comment (codefromthecrypt) [inline trace_context/HTTP_HEADER_FORMAT.md:178]: I think this flag should be removed as it adds little value and results in a hex character "03" which is not intuitive either

File: trace_context/HTTP_HEADER_FORMAT.md (hunk 1 above)
Comment (SergeyKanzhelev) [reply trace_context/HTTP_HEADER_FORMAT.md:150]: "sampled" as a name has a confusion. If "sampled"  means "sampled in" or "sampled out"? The name `requested` eliminates this confusion.

`requested` creates another confusion. Is it `debug` or `sampled`. As specification does not define how sampling probability or sampling rate can be propagated - this confusion is easier to describe in text.

File: trace_context/HTTP_HEADER_FORMAT.md (hunk 2 above)
Comment (SergeyKanzhelev) [reply trace_context/HTTP_HEADER_FORMAT.md:178]: The problem this flag addresses is the ability for vendor to communicate priority of deferred decision. With the semantic "if in doubt - try to collect". See this:

> Caller who defers a tracing decision SHOULD leaves this flag unset. Caller may
communicate the priority of deferred decision (most likely will be collected)
using `recorded` flag.
> Flag `recorded` can also be used to communicate the ask to collect information
or notify that information was collected and available for querying from the
caller service. 

Basically it is a relieve valve to minimize the abuse of `requested` flag.

I understand the problem of `03`, but if we treat those two characters as bitmask - there is very little we can do with this problem unless we are sure there will be no more flags. In fact having two flags not will find out "bad implementations" earlier.

**Comment by codefromthecrypt** [comment]
this does not address sampled. Please get feedback from others not in the
room please

**Comment by SergeyKanzhelev** [comment]
Created https://github.com/w3c/distributed-tracing/issues/167 for discussion

**Comment by SergeyKanzhelev** [comment]
@bogdandrutu @adriancole can you sign off this PR as it just adds clarification for flags introduced in previous PR. Let's discuss renaming and removing in a separate issue.

**Comment by codefromthecrypt** [comment]
I wasn't there at the meeting to say what you've written was clear or not. Not sure bogdan was either. Whoever was there that resulted in this change should ideally be the one clicking approve as they would know.

**Comment by AloisReitbauer** [comment]
I looked at the text and I think the language is still confusing for people. I think we need to cut the text to two things: What the sender intends the receiver to do. How the receiver can change previous decisions.

**Comment by SergeyKanzhelev** [comment]
@AloisReitbauer what people? It is already split into meaning and mutations. Please suggest changes in text

**Comment by SergeyKanzhelev** [comment]
closed in favor of #180

**Pull request closed without merging**